aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -69,6 +69,11 @@ pub fn generate_feature_upgrade_proposal(
 
     let writer = CodeWriter::new(Loc::default());
 
+    emitln!(writer, "// Modifying on-chain feature flags: ");
+    emitln!(writer, "// Enabled Features: {:?}", features.enabled);
```

Comment:
Does this output integer values or labels for enums? Ideally we want labels since integers are not human-readable

Reply:
Enums.